tests/tensorizer_loader/test_tensorizer.py

```diff
@@ -40,21 +46,24 @@
 tensorize_model_for_testing_script = os.path.join(
     os.path.dirname(__file__), "tensorize_vllm_model_for_testing.py")
 
+
 def is_curl_installed():
     try:
         subprocess.check_call(['curl', '--version'])
         return True
     except (subprocess.CalledProcessError, FileNotFoundError):
         return False
 
+
 def get_torch_model(vllm_runner: VllmRunner):
     return vllm_runner \
-            .model \
-            .llm_engine \
-            .model_executor \
-            .driver_worker \
-            .model_runner \
-            .model
+        .model \
+        .llm_engine \
+        .model_executor \
+        .driver_worker \
+        .model_runner \
+        .model
+
```

Comment:
are these changes necessary?

Reply:
Properly appled `format.sh`. Hopefully it looks good now!